.github/workflows/ci.yml

```diff
@@ -755,6 +755,60 @@ jobs:
     - name: Python tests
       run: cmake --build build -t pytest
 
+  win32-debug:
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+        - 3.8
+        - 3.9
+
+        include:
+          - python: 3.9
+            args: -DCMAKE_CXX_STANDARD=20
+          - python: 3.8
+            args: -DCMAKE_CXX_STANDARD=17
+
+    name: "🐍 ${{ matrix.python }} • MSVC 2019 (Debug) • x86 ${{ matrix.args }}"
+    runs-on: windows-2019
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Python ${{ matrix.python }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python }}
+        architecture: x86
+
+    - name: Update CMake
+      uses: jwlawson/actions-setup-cmake@v1.12
+
+    - name: Prepare MSVC
+      uses: ilammy/msvc-dev-cmd@v1.10.0
+      with:
+        arch: x86
+
+    - name: Prepare env
+      run: |
+        python -m pip install -r tests/requirements.txt
+
+    # First build - C++11 mode and inplace
+    - name: Configure ${{ matrix.args }}
+      run: >
+        cmake -S . -B build
+        -G "Visual Studio 16 2019" -A Win32
+        -DCMAKE_BUILD_TYPE=Debug
```

Comment:
I just took a quick look at ci.yml, IIUC we don't have a Windows `Debug` build (only the deadsnakes valgrind build uses `Debug`). I'm mildly surprised, maybe I'm just overlooking something (@henryiii)? If not, adding this `win32-debug` block seems very useful in general, especially because Windows debug builds are fairly tricky in general (from very long-ago past experience). Good to be sure we're covered.

Reply:
Let me reopen the PR then and remove the last commit, so we have coverage?

Reply:
Done